dht.go

```diff
@@ -62,6 +64,7 @@ type IpfsDHT struct {
 	plk sync.Mutex
 
 	protocols []protocol.ID // DHT protocols
+	client    bool
```

Comment:
Although I agree it's not absolutely essential, it would be nice to fix #281 either before or soon after this is merged.

Reply:
https://github.com/libp2p/go-libp2p-kad-dht/issues/281#issuecomment-468608343. Also I should remove this parameter now possibly, I'm not sure.

Reply:
It will be relevant to fix #281, so I'll leave this in.